src/sentry/tasks/post_process.py

```diff
@@ -48,7 +47,7 @@ def _capture_stats(event, is_new):
 
     metrics.incr('events.processed')
     metrics.incr('events.processed.{platform}'.format(platform=platform))
-    metrics.timing('events.size.data', len(six.text_type(event.data)))
```

Comment:
fyi this isnt the same way we compute size in Event.size, but I'm not sure we care